### PR TITLE
Hackathon onboard

### LINKED ISSUE #5039: As an operator of Gitcoin, we'd like to direct hackathon participants from registration right into the Gitcoin onboarding flow.
### User Story
As an operator of Gitcoin, we'd like to direct hackathon participants from registration right into the Gitcoin onboarding flow.

### What 
We want people to register and then immediately see the prizes and get started.

### Why
Reduce friction. And provide better tracking, as having their Gitcoin profile and other associated info in a usable format (ie not separate from other platform info) will help us target people and track their progress through the hackathon flow so we can diagnose where engagement and drop-offs happen for future improvements.

### Assumptions
Human - We assume that hackers will go through this flow.
Business -We assume that directing hackers into one experience vs. multiple experiences (mailchimp, Discord, Gitcoin, etc) will decrease chance of drop off and provide better tracking.
Tech - Use current onboarding flow

### Definition of Done

- [ ] Create a hackathon flow that follows the flow outlined below
- [ ] This flow is to only apply to hackers that come in from the Hackathon landing page.
- [ ] Review with @alexvotofuture @PixelantDesign 

### Data Tracking
- [ ] Add tracking along each step of the funnel based on clicks to the next screen.

### Additional Information
Screen Shot 2019-08-19 at 4 26 43 PM

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

This is all automatically generated when a hackathon explorer is created, admin doesn't need to edit anything.
The onboard guide is the "door" of a hackathon, we want every user new or veteran to go there and understand the hackathon flow. 

**Logged user**

Page dynamically generated for each hackathon explorer following branding. 
image

**Not logged / new user**

Not logged users or new users first logging
image

**Start/work via gitcoin**

Open a modal with more details.
image

**Submit plan plan for hackathon**
The link go to the onboard guide and the message ask for repo and teams usernames
image

**"how to participate" CTA in hackathon explorer**
Link to onboard guide of that hackathon, before people start we want them to understand the hackathon rules
image

**Join button when the hackathon still valid**
If the hackathon date is already expired then hide the onboard link of that hackathon. 
image

##### Refers/Fixes

fix #5039

### CODE REVIEW

**Comment by octavioamu** [comment]
@PixelantDesign @vs77bb @alexvotofuture @willsputra

File: app/dashboard/templates/dashboard/hackathon_onboard.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          {% else %}
+            <div class=" d-flex align-items-center justify-content-around font-bigger-4 text-center text-white">
+              {{ hackathon.name }}
+            </div>
+          {% endif %}
+        <h1>{{hackathon.name}} Guide</h1>
+        {{ hackathon.start_date|date:"M j, Y" }} - {{ hackathon.end_date|date:"M j, Y" }}
+
+      </div>
+      <!-- {% include 'shared/minihero.html' with h1=hackathon.name suppress_logo=1 position_h1_below_logo=True hide_rain=True %} -->
```
Comment (thelostone-mc) [inline app/dashboard/templates/dashboard/hackathon_onboard.html]: do we need this ?

File: app/dashboard/templates/dashboard/hackathon_onboard.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    </div>
+
+    <div class="container">
+
+      {% if hackathon.end_date|timesince >= "1 min" %}
+        <p class="p-3 mt-3 alert-warning">This hackathon event had ended at {{hackathon.end_date}}, please check the <a href="{% url 'get_hackathons' %}">ongoing hackathons</a>.</p>
+      {% endif %}
+      <div class="row py-5">
+        <div class="col-12">
+
```
Comment (thelostone-mc) [inline app/dashboard/templates/dashboard/hackathon_onboard.html]: remove extra line ? :P

File: app/dashboard/templates/dashboard/hackathon_onboard.html [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+<html lang="en">
+  <head>
+    {% include 'shared/head.html' %}
+    {% include 'shared/cards.html' %}
+    <style>
+        .onboard-banner {
+          background: {% firstof hackathon.background_color or 'white' %};
+          color: {% firstof hackathon.text_color or 'black' %};
+        }
+      </style>
```
Comment (thelostone-mc) [inline app/dashboard/templates/dashboard/hackathon_onboard.html]: indent fix?

**Pull request merged**